src/reducers/versions.spec.tsx

```diff
@@ -2837,4 +2880,60 @@ describe(__filename, () => {
       );
     });
   });
+
+  describe('setCurrentVersionId', () => {
+    it('sets the current version ID', () => {
+      const versionId = 42;
+
+      expect(
+        reducer(undefined, actions.setCurrentVersionId({ versionId })),
+      ).toMatchObject({
+        currentVersionId: versionId,
+      });
+    });
+  });
+
+  describe('selectCurrentVersionInfo', () => {
+    it('returns the current version when set', () => {
+      const version = { ...fakeVersion, id: 42 };
+      let state;
+      state = reducer(state, actions.loadVersionInfo({ version }));
+      state = reducer(
+        state,
+        actions.setCurrentVersionId({ versionId: version.id }),
+      );
+
+      expect(selectCurrentVersionInfo(state)).toEqual(
+        createInternalVersion(version),
+      );
+    });
+
+    it('returns null when there was a fetching error', () => {
+      const version = { ...fakeVersion, id: 42 };
+      let state;
+      state = reducer(
+        state,
+        actions.abortFetchVersion({ versionId: version.id }),
+      );
+      state = reducer(
+        state,
+        actions.setCurrentVersionId({ versionId: version.id }),
```

Comment:
Instead of calling `setCurrentVersionId` here, which doesn't really reflect what would happen in the lifecycle of the app, could you instead call `loadVersionInfo` first, which would cause `currentVersionId` to be set, and then call `abortFetchVersion`? That seems like a more natural sequence of events to emulate a fetching error.

Reply:
I made the action dispatching more realistic -- good idea. In this case of a real fetching error, `loadVersionInfo` would never get called, though.